src/main.rs

```diff
@@ -0,0 +1,182 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::{StreamExt};
+use  std::fs;
+use text_io::read;
+
+const TEST_DATABASE: &str = "codeforces-data-model";
+ 
+fn new_core_connection() -> typedb_client::Result<Connection> {
+    Connection::new_plaintext("localhost:1729")
+}
+
+async fn load_data(connection: Connection)->std::io::Result<()>{
```

Comment:
In Rust, it's best if you try to use `.unwrap()` as little as possible. Instead, try to return errors (`Err(...)`). That way your code will crash less. And you can decide what to do if your functions returns an error. 

Your code encounters two types of errors: 
- `std::io::Error` and 
- `typedb_client::error::Error`. 

You can define an enum that wraps both of these errors. When you encounter an error, wrap the error in your enum. An example is here: https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/wrap_error.html

Reply:
Okay sure, thanks for the suggestion!